### PR TITLE
LPS-197548 Content Dashboard Info Panel Perpetually Loads when attempting to open it twice

### PR DESCRIPTION
**Pull request by veroglez**
https://liferay.atlassian.net/browse/LPS-197548

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanel.js [hunk 1]
```diff
@@ -185,12 +185,12 @@ const SidebarPanel = React.forwardRef(
 
 		useImperativeHandle(ref, () => ({
 			close: () => safeDispatch({type: 'CLOSE_SIDEBAR'}),
-			open: (fetchURL, View) => {
+			open: (url, View) => {
 				CurrentViewRef.current = View;
 
-				safeDispatch({type: 'OPEN_SIDEBAR'});
+				safeDispatch({loading: url !== fetchURL, type: 'OPEN_SIDEBAR'});
```
Comment (sandrodw3) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanel.js:191]: Maybe we could just do the `safeDispatch` and the `setFecthURL` if `url` and `fetchURL` are different 😛 It's actually a bit weird to dispatch `'OPEN_SIDEBAR'` with the sidebar already open, what do you think? 😄

**Comment by sandrodw3** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**